cumulusci/tasks/salesforce/ActivateFlowProcesses.py

```diff
@@ -12,28 +13,34 @@ def _init_options(self, kwargs):
         self.options["developer_names"] = process_list_arg(
             self.task_config.options["developer_names"]
         )
```

Comment:
You should be setting a task_options dict in this class to define `developer_names` as a required option for the task. This is used to display help for the options when you run `cci task info activate_flow` and to build the autogenerated reference docs for tasks when you run `cci task doc`